yb-voyager/src/srcdb/mysql.go

```diff
@@ -376,6 +380,10 @@ func (ms *MySQL) GetColumnToSequenceMap(tableList []sqlname.NameTuple) map[strin
 			sequenceName := fmt.Sprintf("%s_%s_seq", tname, columnName)
 			columnToSequenceMap[qualifiedColumeName] = sequenceName
 		}
+		err = rows.Close()
+		if err != nil {
+			utils.ErrExit("close rows for table %s query %q: %s", table.String(), query, err)
+		}
```

Comment:
this is because we need this connection to be free for other parts of code to use?

Reply:
but wouldn't `sql.DB` package should manage in that case and create/maintain a new connection. And if thats the case you can add a warning to log instead of ErrExit??

Reply:
The connection is only released back to the pool after we call `rows.Close()` (https://cs.opensource.google/go/go/+/refs/tags/go1.22.3:src/database/sql/sql.go;l=3424)  . In this case, in every loop iteration, we make a call to the db, so it becomes necessary to close the rows object before the next iteration

Reply:
yeah that i understand, but is `ErrExit` required to close the connection or we can just add a warning in the logs and move forward?

Reply:
I think it would make sense to `ErrExit`, instead of just logging. 
The way I discovered this issue was that the export-data process was stuck (because of this function)
- In first iteration of the for loop, we access db (get a connection from the pool). We do not call rows.Close, and therefore do not return connection back to pool.
- In second iteration of the for loop, it gets stuck while trying to fetch a connection from pool because max open connections = 1.  

Therefore, I think it would make sense to error out.

Reply:
ok, cool. 
Also i think is initial step(before any data export), i was more concerned for cases where whole data and everything is exported but the command fails due to some unimportant error while closing the connection, which can be ignored/retried instead of error out.